GoogleForCreators/web-stories-wp
# Dashboard UI Components: Button & storybook foundations

[Acceptance Criteria](https://github.com/google/web-stories-wp/issues/547)

# Overview 
- Button component for dashboard use. Has primary, secondary, cta variations. 
- Storybook for this button also means some boilerplate to set up storybook on the dashboard side of stories. 
- `"@storybook/addon-actions": "^5.2.8"` package added to simulate actions in storybook. 
- Constants for Dashboard started with button variations (primary and secondary). 
- Theme adds colors, button font, text styling for reuse and eliminates the h1 global font family.

# Testing 
Run `npm run storybook` and see that there's a new section of storybook added. 
- See that the existing stories editor sections are still loading appropriately 
- Navigate to the new Dashboard Button component (http://localhost:56811/?path=/story/dashboard-components-button--default) 
  - See 3 button demos. Default, a "long" button that demos how the button works with overrides from a parent, and a secondary type button. You can use the knobs on any of these three to see how the props work. 

![Mar-13-2020 13-41-57](https://user-images.githubusercontent.com/10720454/76658046-7260bf00-6530-11ea-8f7d-e7b00b108c94.gif)

See #547.
